trollpy/tests.py

```diff
@@ -0,0 +1,357 @@
+        with open(current_place + 'hello.mp3', 'rb') as a:
+            data = a.read()
+        mp3 = Audio(id=1, binary_file=data)
+        dbsession.add(mp3)
+        dbsession.add(kill_score)
+        dbsession.add(user)
+
+# ======== FUNCTIONAL TESTS ===========
+
+
+def test_home_page_pops_up(testapp):
+    """Test that home page get sent correctly."""
+    response = testapp.get('/', status=200)
+    assert response.status_code == 200
+
+
+def test_login_leads_to_home(testapp, fill_the_db):
+    """Test that after logging in it sends you to the home route."""
+    response = testapp.post('/login',
+                            params={'username': 'amos',
+                                    'password': 'password'}).follow()
+    assert response.status_code == 200
+
+
+def test_login_show_form(testapp):
+    """Test that the login route shows form."""
+    response = testapp.get('/login', status=200)
+    assert response.status_code == 200
+    assert response.html.find('form')
+
+
+def test_logout(testapp):
+    """Test that logging out has 200 ok code."""
+    testapp.post('/login',
+                 params={'username': 'amos',
+                         'password': 'password'})
+    response = testapp.get('/logout', status=302).follow()
+    assert response.status_code == 200
+    assert response.html.find_all('a')[0].text == 'log in'
+
+
+def test_registration_view_when_logged_in(testapp, fill_the_db):
+    """Test that registration view when logged in redirects."""
+    testapp.post('/login',
+                 params={'username': 'amos',
+                         'password': 'password'})
+    csrf = testapp.get('/add_smack').html.find_all("input")
+    csrf = csrf[0].attrs['value']
+    response = testapp.get('/registration', params={
+                           'csrf_token': csrf}, status=302).follow()
+    assert 'amosboldor@gmail.com' in response.html.find_all('li')[2].text
+    assert 'Amos Boldor' in response.html.find_all('li')[1].text
+
+
+def test_login_view_when_logged_in(testapp, fill_the_db):
+    """Test that registration view when logged in redirects."""
+    testapp.post('/login',
+                 params={'username': 'amos',
+                         'password': 'password'})
+    csrf = testapp.get('/add_smack').html.find_all("input")
+    csrf = csrf[0].attrs['value']
+    response = testapp.get('/login', params={
+                           'csrf_token': csrf}, status=302).follow()
+    assert 'amosboldor@gmail.com' in response.html.find_all('li')[2].text
+    assert 'Amos Boldor' in response.html.find_all('li')[1].text
+
+
+def test_registration_show_form(testapp):
+    """Test that the registration route shows form."""
+    response = testapp.get('/login', status=200)
+    assert response.status_code == 200
+    assert response.html.find('form')
+
+
+def test_profile_when_not_logged_in(testapp, fill_the_db):
+    """Test that profile works when not logged in."""
+    assert testapp.get('/profile/amos', status=200)
```

Comment:
What content on the front end should I be seeing on the profile page that confirms that I've done the right thing?